libnetwork/cni/network.go

```diff
@@ -295,6 +296,38 @@ func (n *cniNetwork) DefaultInterfaceName() string {
 	return cniDeviceName
 }
 
+// NetworkInfo return the network information about binary path,
+// package version and program version.
+func (n *cniNetwork) NetworkInfo() types.NetworkInfo {
+	path := ""
+	packageVersion := ""
+	for _, p := range n.cniPluginDirs {
+		ver := cutil.PackageVersion(p)
+		if ver != "Unknown" {
```

Comment:
It would make sense to export the `unknownPackage` var and use it here. This will make sure this doesn't regress when someone changes the string.